src/games/strategy/triplea/delegate/Matches.java

```diff
@@ -1928,24 +1877,6 @@ public boolean match(final Unit unit) {
     };
   }
 
-  public static Match<Unit> unitHasEnoughTransportSpaceLeft(final int spaceNeeded) {
```

Comment:
I am surprised this match is not used anywhere. It might be worth keeping `unitHasEnoughTransportSpaceLeft`